pkg/pull/pull.go

```diff
@@ -562,12 +562,18 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		return "", errors.Wrapf(err, "failed to remove unused helm midstreams")
 	}
 
+	renderedKotsKinds, err := kotsutil.KotsKindsFromMap(renderedKotsKindsMap)
```

Comment:
This function was previously only used for tests, but does allow us to leverage the already rendered kots kinds here.